boltzrpc/boltzrpc.proto

```diff
@@ -733,6 +738,39 @@ message GetWalletRequest {
     optional uint64 id = 2;
 }
 
+message ListWalletTransactionsRequest {
+    uint64 id = 1;
+    optional bool exclude_swap_related = 2;
+}
+
+enum TransactionType {
+    ANY = 0;
+    LOCKUP = 1;
+    REFUND = 2;
+    CLAIM = 3;
+    RECEIVE = 4;
+    SEND = 5;
+}
+
+message WalletTransaction {
+    string tx_id = 1;
+    uint64 total_amount = 2;
```

Comment:
Is that the sum of the inputs or outputs?

Reply:
Outputs

Reply:
With out without change output?

Reply:
without change.

Reply:
what about a `net_amount` which is the sum of our own outputs minus the sum of our own inputs?